src/main/kotlin/org/jitsi/nlj/transform/node/PcapWriter.kt

```diff
@@ -90,6 +90,10 @@ class PcapWriter(
     }
 
     fun close() {
+        if (writer.isOpen) {
```

Comment:
`PcapWriter.close` is still only called when the pcap dump is disabled, but not when the `Transceiver` stops (or am I missing something?).

Reply:
i believe i changed that part in my latest commit and factored that code out?